src/libs/SearchUtils.ts

```diff
@@ -52,8 +52,9 @@ function getSections(data: OnyxTypes.SearchResults['data']): SearchTransaction[]
     return searchTypeToItemMap.transaction.getSections(data);
 }
 
-function getQueryHash(query: string): number {
-    return UserUtils.hashText(query, 2 ** 32);
+function getQueryHash(query: string, policyID?: string): number {
+    const textToHash = policyID ? `${query}_${policyID}` : query;
```

Comment:
you can write this as:
`const textToHash = [query, policyID].filter(Boolean).join('_')`;

no conditional logic + whenever we add 3rd or 4th parameter (I believe we will add more params like page or sortBy, right?) it will be easy to expand to more phrases.